### PR TITLE
cli: Update the go-dep version used in the Docker image

### PR DESCRIPTION
**Pull request by sschuberth**
Version 0.5.1-1 is not available anymore, see
https://packages.debian.org/sid/devel/.

Signed-off-by: Sebastian Schuberth <sebastian.schuberth@bosch-si.com>

---
This change is [Reviewable](https://reviewable.io/reviews/heremaps/oss-review-toolkit/1564)

### CODE REVIEW

File: cli/docker/Dockerfile [hunk 1]
```diff
@@ -34,7 +34,7 @@ ENV \
     BOWER_VERSION=1.8.8 \
     BUNDLER_VERSION=1.17.3-3 \
     COMPOSER_VERSION=1.8.4-1 \
-    GO_DEP_VERSION=0.5.1-1 \
+    GO_DEP_VERSION=0.5.1+really0.5.0-1 \
```
Comment (mnonnenmacher) [inline cli/docker/Dockerfile]: Commit message nit: "using" -> "used"

Comment (sschuberth) [reply cli/docker/Dockerfile]: Done.

**Pull request merged**